.github/rd-release-config.yml

```diff
@@ -0,0 +1,167 @@
+filter-by-commitish: False
+include-pre-releases: false
+categories:
+  - title: '🔥 Features'
+    labels:
+      - 'features'
+  - title: '🚀 Enhancemets'
+    labels:
+      - 'performance'
+      - 'enhancement'
```

Comment:
So far we've used the label `enhancement` for new features

Reply:
True, but that makes it difficult to distinguish between just general improvements to older stuff from and actual new features. We can maybe drop the enhancements tag/category.